src/main/java/uk/gov/justice/probation/courtcasematcher/model/offendersearch/MatchType.java

```diff
@@ -1,11 +1,17 @@
 package uk.gov.justice.probation.courtcasematcher.model.offendersearch;
 
 public enum MatchType {
+    /**
+     * This is how offender search describes the search, not used to store offender matches against.
+     */
     ALL_SUPPLIED,
+    NAME_DOB,
```

Comment:
Ah right sorry, had expected maybe there would be a second Enum for the court-case-service model, may have gotten this confused in the chat we were having on Slack. It probably makes sense to split the NAME_DOB into its own Enum so there's a clear distinction between the offender-search interface and the court-case-service interface. Then maybe have a method along the lines of `CourtCaseServiceMatchType.of(MatchType offenderSearchType)` to do the conversion?